geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloClientConfig.scala

```diff
@@ -20,14 +26,19 @@ import scala.util.Try
 /**
  * Client configuration options, loaded from a config file
  *
- * @param instance instance name
- * @param zookeepers zookeeper connect string
- * @param zkTimeout zookeeper timeout
+ * @param config Accumulo client configuration
+ * @param hasInstances instance id or name set in config file
+ * @param instance instance name (not actually used during configuration)
+ * @param zookeepers zookeeper connect string (not actually used during configuration)
+ * @param zkTimeout zookeeper timeout (not actually used during configuration)
  * @param authType  authentication type ("password", "kerberos" or authentication token class)
  * @param principal authentication principal (username, keytab file, etc)
  * @param token authentication token (password, etc)
  */
+@SuppressWarnings(Array("deprecated"))
```

Comment:
What warning is being deprecated?  Is ClientConfiguration deprecated?

Reply:
Yes, ClientConfiguration is deprecated, but because you need to support 1.x I don't think that you can avoid using it without a bunch of gymnastics.